allennlp/tests/fixtures/bidirectional_lm/experiment_unsampled.jsonnet

```diff
@@ -22,7 +22,6 @@
   },
   "model": {
     "type": "bidirectional_language_model",
-    "num_samples": 10,
```

Comment:
Why did this change? I always get paranoid when tests change in a (mostly) refactoring PR... ;)

Reply:
If the experiment is unsampled, isn't the right thing to do to specify a value of `None` to the `"num_samples"`? i.e., i'm pretty sure the `_SoftmaxLoss` wasn't being tested before?

`"num_samples": 10` is being added in `experiment.json` ( https://github.com/allenai/allennlp/blob/master/allennlp/tests/fixtures/bidirectional_lm/experiment.jsonnet#L5 ). Or did I overlook something?

Reply:
Good catch! I added the following to the description:

"Also fixes bug in the experiment_unsampled.jsonnet config that was preventing a test from actually being unsampled."

Even though it's a small fix code-wise, noting the change explicitly could end up being a useful breadcrumb later.